capstone/src/main/webapp/index.html

```diff
@@ -5,17 +5,80 @@
+                Proin id mollis magna
+          </sub>
+        </div>
+        <div class="col l8 m8 s12 push-l1">
+           <h3>This will be a slideshow walkthrough</h3>
+        </div>
+      </div>
+    </section> 
+    <section class="tab-description">
+      <div class="row">
+        <div class="col s12 m12 l12">
+          <ul class="tabs">
+            <li class="tab col s4"><a href="#termsOfUsage">Terms of Usage</a></li>
+            <li class="tab col s4"><a href="#About" >About</a></li>
+            <li class="tab col s4"><a href="#getStarted">Get Started</a></li>
+          </ul>
+          <div class="col s12 tb" id="termsOfUsage">
+            <h3 class="white-text text-darken-4">Terms Of Usage</h3>
+            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pharetra feugiat vehicula. Morbi vitae varius felis.
+                Ut quis interdum dui. Sed pulvinar eros non dictum molestie. Mauris ex arcu, dictum at justo ut, gravida maximus nibh.
+                Proin id mollis magna. Donec ultricies finibus ante, at vestibulum elit volutpat vel. Fusce felis arcu, malesuada finibus 
+                convallis a, vulputate ac tellus. Quisque diam nulla, scelerisque eu mauris ut, vehicula ornare diam.
+            </p>
+          </div>
+           <div class="col s12 tb" id="About">
+            <h3 class="white-text text-darken-4">About</h3>
+            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pharetra feugiat vehicula. Morbi vitae varius felis.
+                Ut quis interdum dui. Sed pulvinar eros non dictum molestie. Mauris ex arcu, dictum at justo ut, gravida maximus nibh.
+                Proin id mollis magna. Donec ultricies finibus ante, at vestibulum elit volutpat vel. Fusce felis arcu, malesuada finibus 
+                convallis a, vulputate ac tellus. Quisque diam nulla, scelerisque eu mauris ut, vehicula ornare diam.
+            </p>
+          </div>
+           <div class="col s12 tb" id="getStarted">
+            <h3 class=" white-text text-darken-4">Get Started</h3>
+            <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pharetra feugiat vehicula. Morbi vitae varius felis.
+                Ut quis interdum dui. Sed pulvinar eros non dictum molestie. Mauris ex arcu, dictum at justo ut, gravida maximus nibh.
+                Proin id mollis magna. Donec ultricies finibus ante, at vestibulum elit volutpat vel. Fusce felis arcu, malesuada finibus 
+                convallis a, vulputate ac tellus. Quisque diam nulla, scelerisque eu mauris ut, vehicula ornare diam.</p>
+          </div>
+        </div>   
+      </div>   
+    </section>   
+    <footer class="page-footer">
+       <div class"container">
+           © 2020 Studentia inc
+       </div>
+    </footer>
      <!--JavaScript at end of body for optimized loading-->
```

Comment:
You have a couple other JS files included at the top of the page but this comment says JS should go at the bottom.